src/ValueObjects/FacetResults.php

```diff
@@ -2,115 +2,94 @@
 
 namespace CultuurNet\SearchV3\ValueObjects;
 
+use JMS\Serializer\Annotation\HandlerCallback;
 use JMS\Serializer\DeserializationContext;
 use JMS\Serializer\JsonDeserializationVisitor;
-use JMS\Serializer\TypeParser;
-use JMS\Serializer\Annotation\HandlerCallback;
 
-class FacetResults implements \Iterator
+final class FacetResults implements \Iterator
 {
-
-    protected $key;
+    /**
+     * @var string|null
+     */
+    private $key;
```

Comment:
I don't think the property `$key` is used.

Reply:
Fixed in https://github.com/cultuurnet/search-v3/pull/43/commits/6502733ec400df3439e5334ca4abb6d5428a47fc